governance/bylaws.md

```diff
@@ -186,11 +189,46 @@ The time and place of every annual or special meeting shall be given to each mem
 A quorum for the transaction of business at any annual or special meeting of members of the Association shall be fifteen (15) Regular or Certified Members in good standing including at least one half of the members of the Board.  If a quorum is not present, the meeting shall be automatically adjourned and the board must convene another meeting within forty five (45) days.
 
 #### 3.4	Voting
-
 All resolutions or motions at an annual or special meeting of members of the Association regarding dissolution, amendments to the Bylaws, or changes to the Code of Ethics shall be approved by a two-thirds majority of the votes cast by Regular or Fellow Members.
 All resolutions or motions at an annual or special meeting of members of the Association regarding dissolution, amendments to the Common Body of Knowledge and Certification/Re-certification Standards shall be approved by a two-thirds majority of the votes cast by Certified Members.
 All other resolutions or motions at an annual or special meeting of members of the Association shall be passed or rejected by a simple majority of the votes cast by Regular or Fellow Members.
-At the discretion of the Board, votes may be cast in person at the meeting, or received by the Secretary by mail or electronic transmission twenty-four (24) hours in advance of the meeting unless otherwise required by the Bylaws of the Association. 
+Votes may be cast by,
+1. indicating Yea/Nay, either in person, by proxy, telephonically or electronically
+2. a secret vote that is,
+   1. marked on a secret ballot cast personally, electronically or by a proxy
+
+##### 3.4.1	Absentee Voting
+Voting by mail or by telephonic or electronic means may only be used if,
+1. the votes may be verified as having been made by members entitled to vote; and
+2. the corporation is not able to identify how each member voted.
```

Comment:
I pulled this clause from https://www.ontario.ca/laws/statute/10n15#BK72 section 67(1).  I included it to indicate exactly what is allowed when it comes to absentee voting (evoting) if we decide to go that way rather than leaving the vague (and possibly illegal) clause we had before.